tests/Feature/Posts/CreatePostsTest.php

```diff
@@ -12,68 +13,135 @@
     {
-        parent::setUp();
-        $this->utility = new Utility($this);
-        $this->utility->testSetup();
+        $this->loginAdmin();
+        $this->postJson(route('api.posts.store'), [
+            'name' => $this->faker->name,
+            'content' => $this->faker->text,
+            'userId' => $this->user->id,
+        ])->assertJsonStructure([
+                'id',
+                'name',
+                'content',
+                'user_id',
+                'status_id',
+                'published_at',
+                'created_at',
+        ]);
     }
 
     /** @test */
     public function it_does_not_allow_non_auth_users()
     {
+        $nonExistingUserId = 0;
         $this->postJson(route('api.posts.store'), [
             'name' => $this->faker->name,
             'content' => $this->faker->text,
-            'userId' => $this->utility->user->id,
+            'userId' => $nonExistingUserId,
         ])
             ->assertStatus(Response::HTTP_UNAUTHORIZED);
     }
 
     /** @test */
     public function it_does_not_allow_guest_to_create_post()
     {
-        $this->utility->loginUser();
+        $this->loginUser();
         $this->postJson(route('api.posts.store'), [
             'name' => $this->faker->name,
             'content' => $this->faker->text,
-            'userId' => $this->utility->user->id,
+            'userId' => $this->user->id,
         ])
             ->assertNotFound();
     }
 
+    /** @test */
+    public function it_brings_back_created_post()
+    {
+        $this->loginAdmin();
+        $name = $this->faker->name;
+        $content = $this->faker->text;
+        $this->postJson(route('api.posts.store'), [
+            'name' => $name,
+            'content' => $content,
+            'userId' => $this->user->id,
+        ])->assertJson([
+            'name' => $name,
+            'content' => $content,
+            'status_id' => Status::DRAFT,
+            'published_at' => null,
+            'user_id' => $this->user->id,
+        ]);
+    }
+
     /** @test */
     public function it_does_allow_auth_users()
     {
-        $this->utility->loginAdmin();
+        $this->loginAdmin();
+        $name = $this->faker->name;
+        $content = $this->faker->text;
+        $this->postJson(route('api.posts.store'), [
+            'name' => $name,
+            'content' => $content,
+            'userId' => $this->user->id,
+        ])->assertOk();
+
+        $this->assertDatabaseHas('posts', [
+            'name' => $name,
+            'content' => $content,
+            'user_id' => $this->user->id,
+            'status_id' => Status::DRAFT,
+            'published_at' => null
+        ]);
+    }
 
+    /** @test */
+    public function it_does_not_allow_non_existing_user()
+    {
+        $this->loginAdmin();
         $name = $this->faker->name;
         $content = $this->faker->text;
+        $nonExistingUserId = User::orderByDesc('id')->first()->id + 1;
```

Comment:
You could probably get away with just using `9999999999999` or something for this ID, that way you arent needing to query the DB but either one works.

Reply:
Honestly forgot to change this when I was refactoring my tests.